commander/test/utils/download.ts

```diff
@@ -12,44 +16,34 @@ describe('download utils', () => {
 
 	describe('#download', () => {
 		let existsSyncStub: any = null;
```

Comment:
i wasn't part of this, no need to use `null`, also we only use `undefined` in the code.
Also, the type can be `SinonStub`